### PR TITLE
Add bugfix for NavMesh import issue

### PR DESCRIPTION
**Pull request by paulbalaji**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
* add text to describe how to bake navmesh in `FPS-SimulatedPlayerCoordinator` scene
* rejig look of the project load bugfixes section
* drive-by typo fixes

#### Tests
local improbadoc

#### Documentation
this is the doc

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

Overall Look:
image

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -157,7 +158,23 @@ There is a quick fix however:
 2. In the Project panel, navigate to **Assets** > **Fps** > **Art** > **Materials**.
 3. Right click on `Source_Shaders` and press Reimport.
 
-<%(#Expandable title="Reimport shaders")%><img src="{{assetRoot}}assets/shader-fix.jpg" style="margin: 0 auto; display: block;" /><%(/Expandable)%>
+<img src="{{assetRoot}}assets/shader-fix.jpg" style="margin: 0 auto; display: block;" />
```
Comment (JonasImprobable) [inline docs/get-started.md]: Is there a raeson to use this HTML-like syntax?

Comment (paulbalaji) [reply docs/get-started.md]: I'm not sure - I'm just using whatever was already in the repo

@samc-improbable ?

Comment (firtina-improbable) [reply docs/get-started.md]: can use `alt` i think

Comment (paulbalaji) [reply docs/get-started.md]: yup I've used alt before, just didn't want to change unless there was a specific reason for putting it like that

Comment (paulbalaji) [reply docs/get-started.md]: reasoning for html style was to ensure images are aligned properly - not so easy to do in MD

Comment (jamiebrynes7) [reply docs/get-started.md]: Yeah it keeps it centered

**Pull request merged**